### PR TITLE
fix linting

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: roles/acme/meta/main.yml [hunk 1]
```diff
@@ -13,3 +13,7 @@ galaxy_info:
     - letsencrypt
     - security
     - ssl
+  platforms:
```
Comment (schurzi) [inline roles/acme/meta/main.yml:16]: I think this one is a fluke. Why did you add this?

Comment (rndmh3ro) [reply roles/acme/meta/main.yml:16]: Because ansible-lint expects it. However I now changed it to a existing value.

**Review by schurzi — APPROVED** [review]
